minecraft_servers/stardix/manifest.json

```diff
@@ -5,36 +5,70 @@
   "server_wildcards": [
     "%.stardix.com"
   ],
+  "supported_languages": [
+    "pt_BR", "pt"
```

Comment:
`pt_BR` isn't part of [ISO 639](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes)

Reply:
I have remove it